spec/Suite/Code/Code.spec.php

```diff
@@ -128,11 +128,11 @@
             };
 
             expect($closure)->toThrow(new TimeoutException('Timeout reached, execution aborted after 1 second(s).'));
-            expect($counter)->toBeGreaterThan(3);
+            expect($counter)->not->toBeLessThan(3);
             expect($counter)->toBeLessThan(6);
 
             $end = microtime(true);
-            expect($end - $start)->toBeGreaterThan(1);
+            expect($end - $start)->not->toBeLessThan(1);
```

Comment:
I will let @jails review about it

Reply:
I suppose the first expectation could be `>= 3`, instead of `> 3`.  And on MacOS that code took exactly `3` - like [in here](https://github.com/thiagodp/kahlan/actions/runs/11421557673/job/31778466807).